sessionctx/variable/session.go

```diff
@@ -1152,6 +1159,19 @@ func (s *SessionVars) WithdrawAllPreparedStmt() {
 	metrics.PreparedStmtGauge.Set(float64(afterMinus))
 }
 
+// SetTmpSystemVar sets the value of a system variable temporarily
```

Comment:
`TmpSystemVar` seems ambiguous, how about renaming `TmpSystemVar` to `SQLLevelVar`? @b41sh @zz-jason

Reply:
sounds good to me.

Reply:
I agree, good idea.